core/tauri/src/manager.rs

```diff
@@ -480,7 +471,7 @@ impl<R: Runtime> WindowManager<R> {
       });
     }
 
-    let window_url = Url::parse(&pending.url).unwrap();
+    let window_url = pending.current_url.lock().unwrap().clone();
     let window_origin =
       if cfg!(windows) && window_url.scheme() != "http" && window_url.scheme() != "https" {
         format!("https://{}.localhost", window_url.scheme())
```

Comment:
Using this change, I had to add  `tauri.localhost` to `dangerousRemoteUrlIpcAccess` for a windows release build to work with the shipped javascript bundle. This didn't seem to be necessary for OSX. This block looks relevant, but haven't dived into the details here.

Reply:
thanks for catching this, I'll take a look.

Reply:
Fix pushed. Tag me if you find anything weird :)